pkg/arrai/model.sysl

```diff
@@ -0,0 +1,34 @@
+Source[~db]:
```

Comment:
Yeah, it'll get there in the end. I've been pushing for the bank model to become our canonical example because it feels a little more serious and familiar. For the time being I'm being deliberately vague.